framework/wazuh/core/config/models/engine.py

```diff
@@ -0,0 +1,37 @@
+from pydantic import BaseModel, FilePath, PositiveInt, PositiveFloat
+from wazuh.core.config.models.logging import LoggingConfig
+
+
+# TODO(#25121): Change the socket path once the Cpp team does it
+class EngineClientConfig(BaseModel):
+    """Configuration for the Engine client.
+
+    Parameters
+    ----------
+    api_socket_path : FilePath
+        The path to the API socket. Default: "/var/wazuh/queue/engine.sock".
+    retries : PositiveInt
+        The number of retry attempts. Default: 3.
+    timeout : PositiveFloat
+        The timeout duration in seconds. Default: 10.0.
+    """
+    api_socket_path: FilePath = "/var/wazuh/queue/engine.sock"
```

Comment:
Use a constant

Reply:
Added in e4e54327c2b3a45cbe77a6f9c30cfd19dc9ce0b6.